pkg/refl/helpers.go

```diff
@@ -215,3 +215,10 @@ func InitializeMapsAndSlices(value interface{}) {
 		}
 	}
 }
+
+func ValueToPointerValue(val interface{}) interface{} {
```

Comment:
Is this really needed when we have `mdl.Box` already?

Reply:
see above